pom.xml

```diff
@@ -796,9 +796,9 @@
                 <!-- Non-standard plugins -->
 
                 <plugin>
-                    <groupId>com.jayway.maven.plugins.android.generation2</groupId>
+                    <groupId>com.simpligility.maven.plugins</groupId>
```

Comment:
Does the android-maven-plugin need to be declared here at all? Previously v3.9.0-rc3 was declared here, but it was actually v4.0.0-rc.1 that was used, and that did not seems to make any difference.